cobalt/loader/net_fetcher.cc

```diff
@@ -124,10 +128,15 @@ NetFetcher::NetFetcher(const GURL& url, bool main_resource,
     request_cross_origin_ = true;
     url_fetcher_->AddExtraRequestHeader("Origin:" + origin.SerializedOrigin());
   }
-  std::string content_type =
-      std::string(net::HttpRequestHeaders::kResourceType) + ":" +
-      std::to_string(options.resource_type);
-  url_fetcher_->AddExtraRequestHeader(content_type);
+
+  if (url.SchemeIsHTTPOrHTTPS()) {
+    auto url_request_context = network_module->url_request_context();
+    // HttpCache::GenerateCacheKey
```

Comment:
I'm not sure what this comment means, can you add more context to it or remove it?